### PR TITLE
Chapter 3 proofreading

### PR DESCRIPTION
**Pull request by psywombats**
Read chapter 3, liking things so far. Comments noted.

### CODE REVIEW

File: chapter03/c03_flan.rpy [hunk 1]
```diff
@@ -11,20 +11,20 @@ label c03_flan:
     He must have thought that I wouldn't do it.
     c "Ehehe... [Pause] Well..."
     c "Here you go! Make sure you eat all of it!"
-    With a smile, I extended the coffered the cup of flan to the stunned boy.
```
Comment (psywombats) [inline chapter03/c03_flan.rpy:14]: I wasn't sure what you meant here.

File: chapter03/c03_flan.rpy [hunk 2]
```diff
@@ -74,35 +74,35 @@ label c03_flan:
...
+            c "Um, Toma? [Pause] Are you alright?"
+            t "Huh?"
+            t "Oh, I'm fine."
+            t "But hey, do me a favor and don't get too close to him, got it?"
+            t "If he keeps on staring at you, let me know and I'll tell the guy to stop."
+            c "Huh? [Pause] Um, well alright."
+            I smiled in confusion at Toma as we continued down the hall, soon talking about what drink to get.
 
     [Transition]
-    It was now six eighteen in the evening, the hospital hallway lights dimmed down to settle a time passing atmosphere.
```
Comment (psywombats) [inline chapter03/c03_flan.rpy:104]: Not sure if you were after figurative language here but I didn't quite understand

File: chapter03/c03_study.rpy [hunk 3]
```diff
@@ -96,81 +93,81 @@ label c03_study:
...
+    I fell silent at this, curious as to why he didn't open them.
     The room is so dark, it's really hard to see. [Pause] So why doesn't he open the curtains?
     I pressed my lips together as I began to glance around the room, feeling immobile for some reason.
     c "...Hm?"
     My wondering eyes stopped as I noticed a tray of food sitting on the ground, in front of the bed.
-    I walked towards the tray, my eyes locked on it- trying to make out every single detail through the darkness.
+    I walked towards the tray, my eyes locked on it - trying to make out every single detail through the darkness.
     c "Hey... [Pause] Did you not eat your lunch? It's sitting here, in front of your bed."
     I knelt down in front of the food to take a better look.
-    The food on the plate consisted of a small portion of rice that had grown cold when you inch your hand near it, a bowl of vegetables and meat, and a cup of warm water that sat in front of three small pills.
```
Comment (psywombats) [inline chapter03/c03_study.rpy:157]: Wasn't quite sure how to work 'when you inch your hand near it' into this sentence so I dropped it

File: chapter03/c03_flan.rpy (hunk 1 above)
Comment (Phantom-Chan) [inline chapter03/c03_flan.rpy:14]: Ah, this is just my hand writing more than what I had thought in my head. I'm okay with, "I extended the cup of flan to the stunned boy."

File: chapter03/c03_flan.rpy [hunk 4]
```diff
@@ -33,7 +33,7 @@ label c03_flan:
     t "Never mind that, why in Ion's name were you not in your room?!"
     I flinched at Toma's stern loud voice, my body tensing up.
     c "I-I'm sorry... [Pause] I just... wanted to welcome the new resident over at room 294..."
-    t "I thought that you were instructed to stay in your room at all times, unless supervised!"
+    t "I thought that you were instructed to stay in your room at all times unless supervised!"
     I looked down at my face, feeling guilty from Toma's angry tone.
```
Comment (Phantom-Chan) [inline chapter03/c03_flan.rpy]: Sorry, "face" should be "feet"

Comment (psywombats) [inline chapter03/c03_flan.rpy]: Oh wow I missed that haha

**Pull request merged**